api.raml

```diff
@@ -0,0 +1,418 @@
+#%RAML 1.0
+title: REST API doc for token.store
+mediaType: application/json
+baseUri: https://v1.api.token.store
+
+types:
+  SortOrder:
+    type: string
+    description: Order
+    enum: [desc, asc]
+  Address:
+    type: string
+    pattern: /^(0x)?[0-9a-f]{40}$/
+/ticker:
+  get:
+    description: Returns 24h trade volume and the last price for each coin.
+    responses:
+      200:
+        body:
+          example: |
+            {
+              "ETH_TTT":{
+                "tokenAddr":"0xf3bc4a5abbb9ffa56c84d27f167e7257c64bd0b2",
+                "quoteVolume":18,
+                "baseVolume":6.050000000000001,
+                "last":0.15,
+                "bid":0.3,
+                "ask":0.15,
+                "percentChange":-90.54688681639148
+              }
+            }
+/trades:
+  get:
+    description: Historical trades Return historical trades executed. Trades returned in ascending order. Option available to query historical trades based on tradeID, default is to return latest 100 trades
+    queryParameters:
```

Comment:
It says "Option available to query historical trades based on tradeID" - but I don't see this option above. /trades/id? Can we add it explicitly?

Reply:
Added